### PR TITLE
Backport of TOC Reorganization - About section

### PR DESCRIPTION
**Pull request by rhomast**

### CODE REVIEW

File: content/en/docs/setup/install/profiles.md [hunk 1]
```diff
@@ -11,11 +11,12 @@ customized as needed.
...
+| Profile           | Description                                                                 | Characteristics
+|-------------------|:----------------------------------------------------------------------------|:-------------
+| `prod`            | Full installation, production configuration.                                | Default profile:<br/>- Full installation.<br/>- Persistent storage. <br/>- Production OpenSearch cluster topology.
+| `dev`             | Development or evaluation configuration.                                    | Lightweight installation:<br/>- For evaluation purposes.<br/>- No persistence.<br/>- Single-node OpenSearch cluster topology.
+| `managed-cluster` | A specialized installation for managed clusters in a multicluster topology. | Minimal installation for a managed cluster:<br/>- Clusters must be registered with an admin cluster to use [multicluster]({{< relref "/docs/about/verrazzanomulticluster" >}}) features.
+| `none`            | A basic configuration is installed. No components are enabled.               | Simplest installation with no components enabled by default.
```
Comment (MelwynPaul) [inline content/en/docs/setup/install/profiles.md:19]: Please remove 'none' row.

**Review by MelwynPaul — COMMENTED** [review]
One comment.

File: content/en/docs/setup/install/profiles.md (hunk 1 above)
Comment (rosemarymarano) [reply content/en/docs/setup/install/profiles.md:19]: That's weird how that got in there; must have been a timing issue.

Comment (MelwynPaul) [reply content/en/docs/setup/install/profiles.md:19]: Yes, even I was surprised.

**Review by MelwynPaul — APPROVED** [review]

**Pull request merged**